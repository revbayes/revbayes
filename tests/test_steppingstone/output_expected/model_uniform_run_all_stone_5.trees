Iteration	Posterior	Likelihood	Prior	psi
0	-21689.46	-21682.49	-6.976766	((Tarsius_syrichta[&index=21]:0.123074,(Hylobates_lar[&index=9]:0.083610,(Pan_paniscus[&index=17]:0.059976,(Colobus_guereza[&index=6]:0.282400,(Chlorocebus_aethiops[&index=5]:0.017803,Macaca_mulatta[&index=13]:0.096095)[&index=42]:0.110468)[&index=25]:0.203094)[&index=32]:0.053517)[&index=30]:0.696674)[&index=36]:0.103602,(((Saimiri_sciureus[&index=20]:0.119770,(Cebus_albifrons[&index=3]:0.213385,Aotus_trivirgatus[&index=1]:0.089411)[&index=26]:0.007208)[&index=33]:0.103681,Callicebus_donacophilus[&index=2]:0.035876)[&index=43]:0.299994,(Daubentonia_madagascariensis[&index=7]:0.108354,((Lepilemur_hubbardorum[&index=11]:0.109752,(((Lemur_catta[&index=10]:0.096707,Varecia_variegata_variegata[&index=22]:0.128484)[&index=40]:0.050793,Propithecus_coquereli[&index=19]:0.089107)[&index=39]:0.034702,(Microcebus_murinus[&index=14]:0.092318,Cheirogaleus_major[&index=4]:0.135434)[&index=35]:0.067179)[&index=38]:0.041223)[&index=27]:0.071351,((Galago_senegalensis[&index=8]:0.150410,Otolemur_crassicaudatus[&index=16]:0.097327)[&index=24]:0.099403,(Loris_tardigradus[&index=12]:0.143966,(Nycticebus_coucang[&index=15]:0.149852,Perodicticus_potto[&index=18]:0.156780)[&index=37]:0.250164)[&index=41]:0.035097)[&index=29]:0.170917)[&index=28]:0.040242)[&index=34]:0.117875)[&index=31]:0.032506,Galeopterus_variegatus[&index=23]:0.177598)[&index=44];
10	-21636.24	-21621.83	-14.41813	(Galeopterus_variegatus[&index=23]:0.471966,((Pan_paniscus[&index=17]:0.082092,((Macaca_mulatta[&index=13]:0.158637,Colobus_guereza[&index=6]:0.217278)[&index=42]:0.025905,Chlorocebus_aethiops[&index=5]:0.082976)[&index=25]:0.135950)[&index=32]:0.033321,Hylobates_lar[&index=9]:0.167293)[&index=30]:0.420435,(Tarsius_syrichta[&index=21]:0.224900,(Daubentonia_madagascariensis[&index=7]:0.259300,((((Microcebus_murinus[&index=14]:0.276600,Cheirogaleus_major[&index=4]:0.068569)[&index=35]:0.049198,Lepilemur_hubbardorum[&index=11]:0.083734)[&index=38]:0.065727,(Propithecus_coquereli[&index=19]:0.129053,(Varecia_variegata_variegata[&index=22]:0.264084,Lemur_catta[&index=10]:0.042388)[&index=40]:0.154041)[&index=39]:0.046243)[&index=27]:0.125023,(((Otolemur_crassicaudatus[&index=16]:0.150019,Galago_senegalensis[&index=8]:0.069985)[&index=24]:0.022268,((Perodicticus_potto[&index=18]:0.166299,Nycticebus_coucang[&index=15]:0.167616)[&index=41]:0.010157,Loris_tardigradus[&index=12]:0.190742)[&index=37]:0.126587)[&index=29]:0.345267,((Saimiri_sciureus[&index=20]:0.244060,(Aotus_trivirgatus[&index=1]:0.066606,Cebus_albifrons[&index=3]:0.197878)[&index=26]:0.008833)[&index=33]:0.154789,Callicebus_donacophilus[&index=2]:0.087666)[&index=43]:0.318952)[&index=34]:0.058036)[&index=31]:0.074875)[&index=28]:0.025012)[&index=36]:0.053550)[&index=44];
