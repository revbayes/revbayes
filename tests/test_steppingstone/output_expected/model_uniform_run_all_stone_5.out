state	power	likelihood
4	0.0529221	-21558.6
5	0.0529221	-21612.3
6	0.0529221	-21634.6
7	0.0529221	-21696.2
8	0.0529221	-21664
9	0.0529221	-21630.6
10	0.0529221	-21621.8
